Specifications/Grpc/for_Hosts/given/ICanBindMyHostType.cs

```diff
@@ -0,0 +1,12 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Dolittle. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+namespace Dolittle.Grpc.given
+{
+    public interface ICanBindMyHostType : ICanBindServices 
```

Comment:
Missing docs

Reply:
We don't put xml docs on classes used in specs - they aren't published anywhere